### PR TITLE
[graphiql] remove unnecessary `<div />` wrappers

### PR DESCRIPTION
**Pull request by B2o5T**

### CODE REVIEW

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 1]
```diff
@@ -507,11 +506,12 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
...
-            {pluginContext?.visiblePlugin ? (
-              <div className="graphiql-horizontal-drag-bar" />
-            ) : null}
-          </div>
+          {pluginContext?.visiblePlugin ? (
+            <div
+              className="graphiql-horizontal-drag-bar"
+              ref={pluginResize.dragBarRef}
+            />
+          ) : null}
```
Comment (B2o5T) [inline packages/graphiql/src/components/GraphiQL.tsx:514]: ## before
image
## now 
image

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 2]
```diff
@@ -571,61 +571,56 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
                       editorContext.tabs.length === 1 ? ' full-height' : ''
                     }`}
                   >
-                    <div ref={editorToolsResize.firstRef}>
-                      <section
-                        className="graphiql-query-editor"
-                        aria-label="Query Editor"
```
Comment (B2o5T) [inline packages/graphiql/src/components/GraphiQL.tsx:577]: ## before
image

## now 
image

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 3]
```diff
@@ -571,61 +571,56 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
...
-                        </div>
-                      </section>
-                    </div>
+                        <ExecuteButton />
+                        {toolbar}
+                      </div>
+                    </section>
                     <div ref={editorToolsResize.dragBarRef}>
                       <div className="graphiql-editor-tools">
-                        <div className="graphiql-editor-tools-tabs">
```
Comment (B2o5T) [inline packages/graphiql/src/components/GraphiQL.tsx:601]: ## before

image

## now 
image

File: packages/graphiql/src/components/GraphiQL.tsx [hunk 4]
```diff
@@ -687,19 +683,18 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
...
-                      editorTheme={props.editorTheme}
-                      responseTooltip={props.responseTooltip}
-                      keyMap={props.keyMap}
-                    />
-                    {footer}
-                  </div>
+                <div
+                  className="graphiql-horizontal-drag-bar"
+                  ref={editorResize.dragBarRef}
+                />
```
Comment (B2o5T) [inline packages/graphiql/src/components/GraphiQL.tsx:689]: ## before

image


## now 
image

**Comment by acao** [comment]
love this. though decreasing DOM depth may seem like microperformance to some, it's important to remember our users want to use graphiql at scale, and often run into these kinds of issues:

https://github.com/graphql/graphiql/discussions/3186

**Comment by B2o5T** [comment]
@acao I guess mentioned issue will be fixed with `monaco-editor` and workers

**Comment by acao** [comment]
@B2o5T hopefully! 

if we can have a pluggable results pane though (I would like this for 4.0.0!), then we can let people use whatever they want to render results

**Pull request merged**